From c1dfe32ce1b721a22475ab458e3f09a2c5b295dd Mon Sep 17 00:00:00 2001
From: Hugues Fruchet <hugues.fruchet@foss.st.com>
Date: Wed, 7 Jun 2023 11:20:44 +0200
Subject: [PATCH 0403/1141] arm64: dts: st: add GPU on stm32mp257f-ev board

Add GPU support on stm32mp257f-ev board

Signed-off-by: Hugues Fruchet <hugues.fruchet@foss.st.com>
Change-Id: Ibf47f91bc3a0dd54cc84245e142e726f67d811e9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/312378
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 625203cf8697..940a64e78438 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -277,6 +277,11 @@ phy1_eth2: ethernet-phy@1 {
 	};
 };
 
+&gpu {
+	contiguous-area = <&gpu_reserved>;
+	status = "okay";
+};
+
 &hpdma {
 	memory-region = <&hpdma1_lli>;
 };
-- 
2.39.2

